Extensions/Ansible/Src/Tasks/Ansible/task.json

```diff
@@ -56,69 +60,175 @@
             "defaultValue": "",
             "required": true,
             "visibleRule": "ansibleInterface = cli",
-            "helpMarkDown": "Remote machine URL"
+            "helpMarkDown": "Ansible machine endpoint"
         },
         {
-            "name": "cliRunOptions",
-            "type": "radio",
-            "label": "Run",
-            "required": true,
-            "defaultValue": "commands",
-            "visibleRule": "ansibleInterface = cli",
+            "name": "playbookSource",
+            "type": "pickList",
+            "label": "Source",
+            "defaultValue": "agentMachine",
             "options": {
-                "commands": "Commands",
-                "script": "Shell script"
+                "agentMachine": "Agent machine",
+                "ansibleMachine": "Ansible machine"
             },
-            "helpMarkDown": "Choose to either run shell commands or a shell script on the remote machine."
+            "required": true,
+            "helpMarkDown": "Select agent machine, if playbook is located on automation agent. In this case playbook is copied to ansible machine. Otherwise, select ansible machine.",

```

Comment:
I feel all this scenario should be captured on Readme file in verbose mode.
Will add it there